### ISSUE TITLE
Add option to collapse map

### ISSUE THREAD

**Issue by azizaparveen** [Esri/solutions-components#587]
### Describe requirements

In the Manager Instant app I would like the ability to configure it to collapse the map when opening.

![Image](https://github.com/Esri/solutions-components/assets/138827632/a45522a1-8bc1-497d-91d9-48a69e134e8e)

We would like to open the application and not show the map since the location is not being tracked actively in our information products. It would be nice if the map was collapsed upon launching.

### Alternatives

Although our feature layer has geographic information, we did not intend to ship with the ability to track location, but rather were going to leave that option up to those deploy.

### Impact

Since the map opens were have been discussing different way to deal with how to manage the location information even though we don't feel that it is relevant in many business cases. When you open the app and the map is blank, it appears that the application is broken, but really it is just a blank map without data since we are intentionally not collecting location.

This configuration could be useful in other applications in our Roadmap, such as Grant Management.

One option could be to use a different tech, like a dashboard.
Another option is to spend time planning a workflow for collecting the location data, even thought it is out of scope.

**Comment by azizaparveen** [comment]
Requested by Brandi, original issue: https://devtopia.esri.com/WebGIS/solutions-web-development/issues/168

We would like to have a app wide configuration option collapse map, which will be off by default

**Comment by azizaparveen** [comment]
https://devtopia.esri.com/WebGIS/arcgis-template-configuration/issues/4449

**Comment by sumitzarkar** [comment]
@azizaparveen 

We have few questions on this requirement:
- This option to collapse the map, needs to be added in the configuration? Please confirm
- When we say collapse map option, does that mean we need to **remove map completely from the app** or **only collapse map** (like we have currently when we expand the information)
- Also, do we need to hide /remove the Map switcher
- Will this option to collapse map will be executed only on app load only once or whenever the map is changed (if map switcher is not removed)

**Comment by azizaparveen** [comment]
@sumitzarkar 

- Yes, it will be a app wide configuration option
- We just need it to be collapsed not removed
- I think we should not hide map switcher, since the layers comes from web map, user won't be able to change to a layer from a different map if needed (downside is it can be confusing for users to see the switcher but not the map, any suggestions @chris-fox? ) 
- I think collapse map should be executed both on app load and map switch event

**Comment by azizaparveen** [comment]
@sumitzarkar IA app team added a config option with a field name of hideMapOnLoad to the configuration. Once the manager component supports the config they will update the app.
This should be in devext by Monday

**Comment by azizaparveen** [comment]
@sumitzarkar The changes in config is in devext now
![image](https://github.com/Esri/solutions-components/assets/138827632/7449701d-0b75-4b05-9ab8-38e701554409)

**Comment by prachichawande-ct** [comment]
@azizaparveen 
## Dev Checklist


Items ​ | Is required​ | Status | Comments​
-- | -- | -- | --
Required NLS changes?​ | No   |   | ​
Required Backward compatibility​ | Yes  |   | ​
Required 508​ | No  |   | ​
Need Sanitize?​ | No  |   | ​
___

## Impact Analysis Report

Impacted Areas | Comments
-- | --
Have Configuration changes?​ | No
Have Runtime changes?​ | Yes
Impact on RTL | No
Impact at runtime | On app load, the map should be collapsed in the following cases:
||<u><li>If the config value for hide map on load is true then the map should be collapsed and vice versa</li><li>On map change, if the config value for hide map on load is true then the map should be collapsed and vice versa</li></ul>
||For Backward compatibility, the hide map on load option should be disabled|

**Comment by azizaparveen** [comment]
@sumitzarkar Hide map is not being honored and map is not collapsing, could you please check if the code is in devext, Test app: https://holistic.mapsdevext.arcgis.com/apps/instant/manager/index.html?appid=35fdba35cc1d456a9ced14147068a5ef, 
template issue link: https://devtopia.esri.com/WebGIS/arcgis-portal-app-templates/issues/10005

**Comment by azizaparveen** [comment]
It needed an update from IA side, John made the change, it is working with the Hide map parameter

**Comment by sumitzarkar** [comment]
@azizaparveen 

> in the config preview when I hide the map the preview do not get updated

@jmhauck 

Do we need to watch the poroprty change for this or this needs to be fixed in Instant app?

Please let us know if this needs any changes in manager component

**Comment by jmhauck** [comment]
@sumitzarkar I added a watch but forgot about the difference in mobile so we'll still need to handle for mobile
